README.md

```diff
@@ -108,6 +108,14 @@ Default: null
 Allows you to specify the Html runner that Jasmine uses **only** during
 integration tests.
 
+#### phantomjsPath
+Type: `string` <br />
+Default: phantomjs
+
+Allows you to specify the path to your phantomjs installation. After 
+`npm install phantomjs --save-dev` you can set the path to `./node_modules/phantomjs/lib/phantom/bin/phantomjs`
```

Comment:
Instead of limiting the use case of only installing `phantomjs` via `--save-dev` it is better to simply state you can provide a string path. Many times people can have different versions of Phantom installed and the path is not necessarily based off of the `npm install`.